plugins/color_panel/.gitignore

```diff
@@ -1,3 +1,8 @@
 .packages
 .flutter-plugins
 pubspec.lock
+.flutter-plugins-dependencies
```

Comment:
Hey @stuartmorgan, what are your feelings on adding these entries to the individual `.gitignore` files?

Reply:
I'd much rather Flutter not spew files into plugin directories.

It'll be irrelevant once the recommended way to consume these plugins isn't by path. Since that's a hack anyway, I'm fine with either compensating with this, or ignoring the problem until it goes away. As noted previously, the latter has been my approach so far.

Reply:
Ok I'll take the ignore it until it goes away approach.